visualization/app/codeCharta/codeCharta.component.ts

```diff
@@ -8,10 +8,13 @@ import { InjectorService } from "./state/injector.service"
 import { StoreService } from "./state/store.service"
 import { setAppSettings } from "./state/store/appSettings/appSettings.actions"
 import { setIsLoadingFile } from "./state/store/appSettings/isLoadingFile/isLoadingFile.actions"
-import * as codeCharta from "../../package.json"
+import packageJson from "../../package.json"
 import { setDelta, setMultiple, setSingle } from "./state/store/files/files.actions"
 import { getCCFiles } from "./model/files/files.helper"
+import sample1 from "./assets/sample1.cc.json"
+import sample2 from "./assets/sample2.cc.json"
```

Comment:
Note: what I personally do in situations like these is to safe the object in a `.ts` file and mark the object as e.g., `ExportCCFile`. That way you have the type safety while still being able to import it easily.

Reply:
We'd have two files with the same content then though. We can't just remoe the `.cc.json` files. :(

Reply:
Yes, but right now we don't have two identical json files.

Reply:
Why would the TS Files be identical in that case?

Reply:
Depending on the cc.json it would be quite long. The only difference would be something like:
```ts
const json : CCFile = {
...MY_REALLY_LONG_JSON...
}
```

Reply:
I'm confused... I am not certain why you would want to keep both?

Reply:
I think we might use the `.json` files for testing purposes and for the url-parameters. But I have to take a deeper look.

Reply:
Right, I see. Thanks!